server/app/services/applicant/question/AddressQuestion.java

```diff
@@ -269,6 +270,20 @@ public Path getServiceAreaPath() {
     return serviceAreaPath = applicantQuestion.getContextualizedPath().join(Scalar.SERVICE_AREA);
   }
 
+  public Address getAddress() {
+    return Address.builder()
+        .setStreet(getStreetValue().orElse(""))
+        .setLine2(getLine2Value().orElse(""))
+        .setCity(getCityValue().orElse(""))
+        .setState(getStateValue().orElse(""))
+        .setZip(getZipValue().orElse(""))
+        .build();
+  }
+
+  public Boolean needsAddressCorrection(Boolean isAddressCorrectionEnabled) {
+    return isAddressCorrectionEnabled && getCorrectedValue().isEmpty();
```

Comment:
**suggestion (blocking):** please add javadoc for this public method

Reply:
done